### PR TITLE
Doc 785 agents chains

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/advanced-ai/examples/understand-chains.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: What is a chain in AI?
+description: Understand chains in the context of AI. Learn about chains in n8n.
+contentType: explanation
+---
+
+# What is a chain in AI?
+
+Chains bring together different components of AI (such as models and memory) to create a cohesive system. They set up a sequence of calls between the components.
```
[the remark this answers is not in the document]
Comment (ayatnkw) [reply docs/advanced-ai/examples/understand-chains.md:9]: I interpreted the text as describing chains as a general concept in AI that typically include memory support. But maybe we could clarify this sentence to make that clear?

Comment (StarfallProjects) [inline docs/advanced-ai/examples/understand-chains.md:9]:
```suggestion
Chains bring together different components of AI to create a cohesive system. They set up a sequence of calls between the components. These components can include models and memory (though note that in n8n chains can't use memory).
```

[the remark this answers is not in the document]
Comment (StarfallProjects) [reply docs/advanced-ai/examples/understand-chains.md]: Thanks! Aya's interpretation is correct, but I've clarified.

File: docs/advanced-ai/examples/introduction.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ This section provides explanations of important AI concepts, and workflow templa
 
 <div class="grid cards" markdown>
 
+-   __Agents and chains__
+
+	Learn about agents and chains in AI, including exploring key differences using the example workflow.
+
+	[:octicons-arrow-right-24: What is a chain in AI?](/advanced-ai/examples/understand-chains/)  
+    [:octicons-arrow-right-24: What is an agent in AI?](/advanced-ai/examples/understand-agentss/)
```
Comment (OlegIvaniv) [inline docs/advanced-ai/examples/introduction.md:18]: I think there's a typo here in the url(double s)

File: docs/advanced-ai/examples/understand-chains.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Chains bring together different components of AI to create a cohesive system. They set up a sequence of calls between the components. These components can include models and memory (though note that in n8n chains can't use memory).
+
+
+## Chains in n8n
+
+n8n provides three chain nodes:
+
+* [Basic LLM Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainllm/): use to interact with an LLM, without any additional components.
+* [Retrieval Q&A Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainretrievalqa/): can connect to a vector store using a retriever. Use this if you want to create a workflow that supports asking questions about specific documents.
```
Comment (OlegIvaniv) [inline docs/advanced-ai/examples/understand-chains.md:17]: Might be worth mentioning here that you can also connect Workflow Retriever to fetch context using workflow

**Review by OlegIvaniv — APPROVED** [review]
Thank you for addressing my points, LGTM.

**Pull request merged**